source/includes/steps-starting-compass.yaml

```diff
@@ -15,7 +15,13 @@ content: |
        * - :guilabel:`Hostname`
 
          - The hostname of the machine where the ``mongod`` instance is
-           running.
+           running. 
+
+           .. tip::
+
+              Compass automatically populates other fields in this dialog
```

Comment:
@kay-kim , it's not clear from your comment what you didn't like about the prior sentence.  I'm guessing that you wanted the dependent clause to come first? I just cut-and-pasted your sentence, but made "field" plural.  Staged at https://docs-mongodbcom-staging.corp.mongodb.com/compass/robertjustice/DOCSP-870/connect.html and ready for another look.